src/odemis/odemisd/modelgen.py

```diff
@@ -679,6 +679,12 @@ def _instantiate_comp(self, name):
         if "power_supplier" in args:
             f = args["power_supplier"].supply({name: True})
             f.result()
+        # Also turn on the children
```

Comment:
Is it always safe to turn on these components?

Reply:
Usually it is. But, in any case, that's the responsibility of the person writing the microscope file. In the microscope file, it is said "to be able to use this component, it must first be turned on using this power supplier". So this new code just ensures that we follow the rules laid down in the microscope file.